content/SCALE/SCALETutorials/Apps/CommunityApps/jellyfin.md

```diff
@@ -0,0 +1,172 @@
+
+You can allow SCALE to create the datasets Jellyfin requires automatically during app installation.
+Or before beginning app installation, [create the datasets]({{< relref "DatasetsSCALE.md" >}}) to use in the **Storage Configuration** section during installation.
+Jellyfin requires two datasets: **config** and **cache**.
+You can organize these as one parent with two child datasets, for example <file>mnt/tank/jellyfin/config</file>, <file>mnt/tank/jellyfin/cache</file>, and so on.
+You can choose to create a static **transcodes** dataset or use temporary storage in the disk or memory for transcoding.
+
+## Installing the Jellyfin Application
+
+To install the **Jellyfin** application, go to **Apps**, click **Discover Apps**, either begin typing Jellyfin into the search field or scroll down to locate the **Jellyfin** application widget.
+
+{{< trueimage src="/images/SCALE/Apps/JellyfinWidget.png" alt="Jellyfin App Widget" id="Jellyfin App Widget" >}}
+
+Click on the widget to open the **Jellyfin** application details screen.
+
+{{< trueimage src="/images/SCALE/Apps/JellyfinAppDetailsScreen.png" alt="Jellyfin App Details Screen" id="Jellyfin App Details Screen" >}}
+
+Click **Install** to open the Jellyfin application configuration screen.
+
+Application configuration settings are presented in several sections, each explained below.
+To find specific fields click in the **Search Input Fields** search field, scroll down to a particular section or click on the section heading on the navigation area in the upper-right corner.
+
+{{< trueimage src="/images/SCALE/Apps/InstallJellyfinScreen.png" alt="Install Jellyfin Screen" id="Install Jellyfin Screen" >}}
+
+Accept the default values in **Application Name** and **Version**.
+
+You can enter a **Published Server URL** for use in UDP autodiscovery or leave this blank.
+
+Enter the desired User and Group IDs to run the Jellyfin container in **User and Group Configuration** or accept the default 568 (apps) user and group.
+
+Accept the defaults in **Network Configuration** or change to suit your use case.
+
+Jellyfin requires two storage datasets.
+You can allow SCALE to create them for you, or use the dataset(s) created in [First Steps](#first-steps).
+Select the storage options you want to use for **Jellyfin Config Storage** and **Jellyfin Cache Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset or select **Host Path** to use the existing datasets created on the system.
+
+Jellyfin also requires a dataset or emptyDir for **Jellyfin Transcodes Storage**.
+Select **ixVolume (dataset created automatically by the system)** in **Type** to let SCALE create the dataset, select **Host Path** to use an existing dataset created on the system, or select **emptyDir** to use a temporary directory on the disk or in memory.
+
+Accept the defaults in Resources or change the CPU and memory limits to suit your use case.
```

Comment:
resources should be lower case here or use bold and the full section name **Resource Configuration**
```suggestion
Accept the defaults in resources or change the CPU and memory limits to suit your use case.
```